modules/140-user-authz/crds/clusterauthorizationrule.yaml

```diff
@@ -212,6 +212,7 @@ spec:
                     If the `namespaceSelector` parameter is not specified, then the set of available namespaces is determined by the value of the `limitNamespaces` and `allowAccessToSystemNamespaces` parameters. If none of the parameters `namespaceSelector`, `limitNamespaces` and `allowAccessToSystemNamespaces` are specified, then all namespaces will be available, except for the system ones (`kube-*`, `d8-*`, `loghouse`, `default`).
 
                     Is available **only** if the [enableMultiTenancy](configuration.html#parameters-enablemultitenancy) parameter is enabled.
+                    **Available in Enterprise Edition only.**
```

Comment:
You should use `x-doc-d8Revision: ee` like for the other EE-only options instead of adding a comment manually.

Reply:
Fixed